### DISCUSSION TITLE
Suppress routine stdout messages for snapshot-only mode

### DISCUSSION THREAD

**Discussion by rmaeder** [psy0rz/zfs_autobackup#202] [Q&A] [answered]
I am trying to use zfs-autobackup to create snapshots on a regular basis (no backups). The docs say to leave out the target argument to do so. In my case it looks like this:
`zfs-autobackup tag`
where I tagged the desired datasets with _autobackup:tag_

When I run this I always get a message on stdout:
`NOTE: Source and target are on the same host, excluding received datasets from selection.`
which is a bit strange, as I didn't even specify a target.

If I do this in a cronjob, this stdout message triggers an email alert.
I can redirect stdout to /dev/null to prevent these emails, but I wonder whether there is a way to suppress the message in the first place.

**Comment by psy0rz** [comment, marked as answer]
can you try this with 3.2?

**Reply by rmaeder** [reply]
With zfs-autobackup v3.2-beta1 there are no more messages produced by my crontab entry; redirection of stdout is no longer necessary.
